MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -74,15 +72,15 @@ - (NSArray *)batchFromEvents:(NSArray *)events {
 }
 
 - (void)postEvents:(NSArray *)events completionHandler:(nullable void (^)(NSError * _Nullable error))completionHandler {
-    [self.metricsManager updateMetricsFromEventQueue:events];
+    [MMEMetricsManager.sharedManager updateMetricsFromEventQueue:events];
```

Comment:
Any reason why you're switching to accessing the singleton here rather than via the property?